content/rs/databases/configure/eviction-policy.md

```diff
@@ -36,10 +36,12 @@ To prevent this from happening, make sure your database is large enough to hold
 
 `volatile-lru` is the default eviction policy for most databases.
 
-For performance reasons, the default policy for [Active-Active databases]({{< relref "/rs/databases/active-active/_index.md" >}}). 
+The default policy for [Active-Active databases]({{< relref "/rs/databases/active-active/_index.md" >}}) is _noeviction_ policy.
 
 ## Active-Active database eviction
 The eviction policy mechanism for Active-Active databases kicks in earlier than for regular databases because it requires propagation to all participating clusters. The eviction policy starts to evict keys when one of the Active-Active instances reaches 80% of its memory limit. If memory usage continues to rise while the keys are being evicted, the rate of eviction will increase to prevent reaching the Out-of-Memory state.
+As with standalone Redis databases, Active-Active eviction is calculated per shard.
+To prevent over eviction, internal heuristics might prevent keys from being evicted when the shard reaches the 80% memory limit.  In such cases, keys will get evicted only when shard memory 100%.
```

Comment:
```suggestion
To prevent over eviction, internal heuristics might prevent keys from being evicted when the shard reaches the 80% memory limit.  In such cases, keys will get evicted only when shard memory reaches 100%.
```